### PR TITLE
[RNMobile] BottomSheet controls

### PR DESCRIPTION
**Pull request by jbinda**
## Description
PR is connected with [#1365](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1365) in gutenberg-mobile.

Please also refer to:

[Related gutenberg-mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/1380)
merge this PR first [[RNMobile] add RangeControl mobile implementation](https://github.com/WordPress/gutenberg/pull/17282) and [[RNMobile] refactor InspectorControls](https://github.com/WordPress/gutenberg/pull/17279)

It presents ability to select all parents in cascade way (according to the way how selecting in groups works on web version)

## How has this been tested?
Manual test on Android and iOS plus CI tests

## Types of changes
Adds new layer on top of `BottomSheet.Cells` to use controls placed in `BottomSheet` in the same way as `Sidebar` controls in web version.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate.

### CODE REVIEW

**Comment by jbinda** [comment]
@etoledom I have open PR to review and target to `master` branch, can you take a quick look once again ?

File: packages/components/src/select-control/index.native.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+// import { withInstanceId } from '@wordpress/compose';
```
Comment (etoledom) [inline packages/components/src/select-control/index.native.js]: We should remove the commented code 👍 
Same on `text-control/index.native.js` and `toggle-control/index.native.js `

**Review by etoledom — APPROVED** [review]
Nice job @jbinda !

Tested `TextControl`, `ToggleControl` and `SelectControl` on both iOS and Android and they just work beautifully 🎉 

**Note:** Added a comment on code to be fixed before merging, but overall looks great.

Regarding the failing test, it's a known issue that a test regarding autosave might fail. https://github.com/WordPress/gutenberg/pull/17652#issuecomment-536944134

File: packages/components/src/select-control/index.native.js (hunk 1 above)
Comment (jbinda) [reply packages/components/src/select-control/index.native.js]: done ! :)

**Comment by etoledom** [comment]
I re-triggered CI (a few times) now they are all passing 🎉

**Comment by jbinda** [comment]
thanks ! shall we merge it ? :)

**Pull request merged**